openapi/components/schemas/Usage.yaml

```
@@ -0,0 +1,45 @@
+type: object
+required:
+  - subscriptionId
+  - planId
+  - quantity
+properties:
+  subscriptionId:
+    description: ID of the subscription for which a usage is reported.
+    allOf:
+      - $ref: ./ResourceId.yaml
+  planId:
+    description: ID of the plan for which a usage is reported.
+    allOf:
+      - $ref: ./ResourceId.yaml
+  invoiceId:
+    description: ID of the invoice for which usage is considered. This value is added when the invoice with usage is issued.
```

I feel this description should be more prescriptive. Maybe:
```suggestion
    description: ID of the invoice to apply usage. Use this field to add usage to an issued invoice.
```

---

I seemed to miss the read only nature of this and the next property. I adjusted my suggestion.

---

Suggestion to avoid ending the first sentence with preposition 
(incorporates Adam's comment )
```suggestion
    description: ID of the invoice to which usage is applied. This value is populated when the invoice is issued.
```